pact.cabal

```diff
@@ -165,7 +165,7 @@ library
                      , hspec >= 2.2.4 && < 2.8
                      , lens >= 4.14 && < 4.18
                      , lens-aeson >= 1.0.0.5 && < 1.1
-                     , megaparsec >= 6
+                     , megaparsec >= 7
```

Comment:
`neat-interpolation` seems to not have been given bounds in the two components it's used in here, but it would now be `>= 0.3.2.2` to account for the strict `megaparsec` lower bound.